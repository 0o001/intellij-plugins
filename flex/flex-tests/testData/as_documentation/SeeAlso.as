package {

/**
 * @see #method()
 * @see #prop1
 * @see http://www.google.com
 * @see flash.ui.ContextMenu#clone()
 * @see "Some text"
 * @see http://livedocs.adobe.com/flex/3/html/help.html?content=asdoc_7.html Adobe documentation of @see tag
 * @see TopLevelClass this is me!

 *
 */
public class TopLevel<caret>Class {

    /**
     * Method
     */
    public function method() {}

    /**
     * Prop
     */
    public var prop1 : String;

}

}