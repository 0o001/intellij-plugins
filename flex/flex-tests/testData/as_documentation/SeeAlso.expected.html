<DL><DT><b>See also:</b></DT><DD><a href="psi_element://TopLevelClass.method()">#method()</a></DD><DD><a href="psi_element://TopLevelClass.prop1">#prop1</a></DD><DD><a href="http://www.google.com">http://www.google.com</a></DD><DD><a href="psi_element://flash.ui.ContextMenu.clone()">flash.ui.ContextMenu#clone()</a></DD><DD>Some text</DD><DD><a href="http://livedocs.adobe.com/flex/3/html/help.html?content=asdoc_7.html">Adobe documentation of @see tag</a></DD><DD><a href="psi_element://TopLevelClass">this is me!</a></DD></DL>